app/(auth)/signin/action/index.ts

```diff
@@ -2,7 +2,7 @@
 import { cookies } from 'next/headers'
 import { redirect } from 'next/navigation'
 import type { SignInSchema } from '@/(auth)/signin/schema'
-import { createClient } from '@/(auth)/supabase/server'
+import { createClient } from '@/(auth)/supabase/with-cookie'
```

Comment:
This is just question! What is the reason to separate with `with-cookie` and `with-request`? Is there any downside to combine together?